packages/os/etc-systemd-network.mount

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=systemd-networkd configuration directory
+DefaultDependencies=no
+Conflicts=umount.target
+Before=generate-network-config.service
```

Comment:
Need to add the standard mount unit dependencies:
```suggestion
Before=local-fs.target umount.target generate-network-config.service
```

But I'd really prefer that we used `RequiresMountsFor` in `generate-network-config.service` instead. Consider doing that in a drop-in for that unit that's added for `%{with systemd_networkd}`.